manifests/h/Hamdpac/gissaettnummer/1.0.0/Hamdpac.gissaettnummer.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Hamdpac.gissaettnummer
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/Hamdpac/gissa_ett_nummer/releases/download/1.0.0/setup.exe
+  InstallerSha256: 21F388D05AFC2049EAA99AF65510ABE2D9DAC72D71C97498EAD0DDE076E90A77

```

```suggestion
  InstallerSha256: 21F388D05AFC2049EAA99AF65510ABE2D9DAC72D71C97498EAD0DDE076E90A77
  AppsAndFeaturesEntries:
  - DisplayName: gissa ett nummer version 0.5
    DisplayVersion: 0.5
    ProductCode: '{9444E378-0DC6-4143-9A9B-F4EE6E287BDE}_is1'
```
Since the version in control panel is different than the GitHub release number, AppsAndFeaturesEntries are needed